website/_sass/_base.scss

```diff
@@ -0,0 +1,233 @@
+/**
```

Comment:
Did u intend to have the .scss files under "_scss" directory? Perhaps a typo in the dir name: "_sass" -> "_scss" ?